src/collectd.conf.pod

```diff
@@ -5166,6 +5166,25 @@ statistics available. Defaults to B<false>.
 
 =back
 
+=head2 Plugin C<nfs>
+
+The I<nfs plugin> collects information about the usage of the Network File
+System (NFS). It counts the number of procedure calls for each procedure,
+grouped by version and whether the system runs as server or client.
+
+It is possibly to omit metrics for a specific NFS version by setting one or
+more of the following options to B<true> (all of them default to B<false>). 
+
+=over 4
+
+=item B<IgnoreV2> B<false>|B<true>
```

Comment:
We avoid using negated config options, because the resulting double-negations ("ignore: false!") are unnecessarily hard to read. The typical naming for such options is "Report…", i.e. in your case `ReportV2`, etc.